### DISCUSSION TITLE
sergen Oracle.ManagedDataAccess.Client error

### DISCUSSION THREAD

**Discussion by wow-prime** [serenity-is/Serenity#6475] [Help]
There has error when i use "dotnet sergen g" to generate code.
My Serenity Code Generator version is 5.2.3.0.
and error message is below:
![image](https://user-images.githubusercontent.com/19904096/182343961-4329f7e5-b85f-404e-8660-b495f0d18153.png)

My appsettings.json is below:
![image](https://user-images.githubusercontent.com/19904096/182344293-a1c96c82-80be-435a-9c3e-172711d1dcfc.png)

How can I solve this issue?

**Comment by wesley-huang-promise** [comment]
@wow-prime Did you find a solution to this issue? I ran into the same problem today.

**Reply by wow-prime** [reply]
No.
If your database is Oracle, it can't use "dotnet sergen g" to generate code.
So I generate code by myselft.

**Reply by hamadaDev** [reply]
how ?

**Reply by wow-prime** [reply]
Copy and Past.
Modify namespace, controller name etc.

**Comment by hamadaDev** [comment]
I have Same Error
with .net core 6 any  one have solution
